client/blocks/product-selector/index.jsx

```diff
@@ -146,22 +146,32 @@ export class ProductSelector extends Component {
 		);
 	}
 
-	getProductSlugByCurrentPlan() {
+	getProductSlugsForCurrentPlan() {
 		const { currentPlanSlug, productSlugs } = this.props;
+		return ! currentPlanSlug
+			? null
+			: filter( productSlugs, productSlug => planHasFeature( currentPlanSlug, productSlug ) );
+	}
 
-		if ( ! currentPlanSlug ) {
-			return null;
-		}
+	currentPlanIncludesProduct( product ) {
+		const { currentPlanSlug } = this.props;
+		const productSlugs = this.getProductSlugsForCurrentPlan();
+		return ! currentPlanSlug || ! productSlugs
+			? false
+			: productSlugs.some( slug => product.slugs.includes( slug ) );
+	}
 
-		return find( productSlugs, productSlug => planHasFeature( currentPlanSlug, productSlug ) );
+	getProductSlugByCurrentPlan( product ) {
```

Comment:
Knowing the code, this is easy to read, but if I was looking this for the first time, it would be a bit confusing - "get the product slug by current plan of the passed product" - feels weird without context. I wonder if we should refactor these, now that they will be taking care of more than one product.

Reply:
It was confusing when I first saw it and it's still confusing now 😆. We ought to consider refactoring for clarity in the future.